.github/dependabot.yml

```diff
@@ -1,10 +1,20 @@
-# To get started with Dependabot version updates, you'll need to specify which
-# package ecosystems to update and where the package manifests are located.
-# Please see the documentation for all configuration options:
-# https://docs.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
-
+# See https://docs.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
 version: 2
 updates:
+  - package-ecosystem: "npm"
+    directory: "/"
```

Comment:
❓ To avoid to scan everything, we can scan only the directory where the projects are ?

Reply:
good catch 👍🏿 I am changing this

Reply:
done in eadb5078ed2bb581420f6936ac47ff504691186f